### ISSUE TITLE
Configurable ini

### ISSUE THREAD

**Issue by vogues** [praydog/RE2-Mod-Framework#61]
I don't have insert on my keyboard, a way to save settings and change them would be greatly appreciated.

Perhaps an ini file that I can set to read only when I've locked in settings.

**Comment by anzbert** [comment]
yes same problem. i am using an apple keyboard on my windows machine and there is no insert button unfortunately.
